File: Serene/Serene.Script/Imports/FormContexts/Meeting.MeetingTypeForm.cs

namespace Serene.Meeting
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;
    using Serene.Northwind;

    public partial class MeetingTypeForm : PrefixedContext
    {
        public MeetingTypeForm(string idPrefix) : base(idPrefix) {}
    
        public StringEditor Name { get { return ById<StringEditor>("Name"); } }
        public StringEditor IsActive { get { return ById<StringEditor>("IsActive"); } }
    }
}

File: Serene/Serene.Core/Imports/ServerTypings/Meeting.MeetingTypeForm.ts
namespace Serene.Meeting {
    export class MeetingTypeForm extends Serenity.PrefixedContext {
        static formKey = 'Meeting.MeetingType';

    }

    export interface MeetingTypeForm {
        Name: Serenity.StringEditor;
    }

    [['Name', () => Serenity.StringEditor]].forEach(x => Object.defineProperty(MeetingTypeForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

